src/res/translation/translation_pl_PL.ts

```diff
@@ -3206,57 +3206,57 @@ Twój kanał został wyciszony i włączono „Wycisz mnie”. Napraw przyczynę
     <message>
         <location filename="../../serverdlg.cpp" line="125"/>
         <source>Recording has been switched off by the UI checkbox</source>
-        <translation type="unfinished"></translation>
+        <translation>Nagrywanie zostało wyłączone przez użytkownika</translation>
     </message>
     <message>
         <location filename="../../serverdlg.cpp" line="127"/>
         <source>Recording has been switched off, either by the UI checkbox or SIGUSR2 being received</source>
-        <translation type="unfinished"></translation>
+        <translation>Nagrywanie zostało wyłączone przez użytkownika lub otrzymano komunikat SIGUSR2</translation>
     </message>
     <message>
         <location filename="../../serverdlg.cpp" line="148"/>
         <source>Language</source>
-        <translation type="unfinished">Język</translation>
+        <translation>Język</translation>
```

Comment:
```suggestion
        <translation>Język (Lang)</translation>
```

I think @BLumia and I talked about adding (Lang) to the box to ensure people still understand it if the selected language is unknown

Reply:
I don't understand from where those 'unfinished' entries came. Either QtLinguist shows all done and text editor search finds none.

Reply:
Do you mean in @ann0see's comment above? That's just the diff. The red part shows what it looked like before your changes (= what has been removed), and the green part shows the new parts. So everything is fine. :)

@ann0see Would it make sense to solve this `(lang)` thing in the code instead of asking all translators to add it? Or would you only add it for translations which don't contain `*Lang*`?

Reply:
Probably adding it to the code would be good; however this would also show it in English then.